checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitVisitor.java

```diff
@@ -811,7 +814,26 @@ public Void visitDoWhileLoop(DoWhileLoopTree tree, Void p) {
     @Override
     public Void visitConditionalExpression(ConditionalExpressionTree tree, Void p) {
         checkForNullability(tree.getCondition(), CONDITION_NULLABLE);
-        return super.visitConditionalExpression(tree, p);
+        // Note: A copy of condThen is provided as an argument to the second invocation of
```

Comment:
How about: `Because of the hack in NullnessVisitor#commonAssignmentCheck, this code needs to use a copy of the types for the two invocations of #commonAssignmentCheck. These hacks should be undone, by properly resolving polymorphic qualifiers in the ATF/transfer.
As this code is mostly duplicated from the super method, this code needs to be kept in sync.`